tools/cleanup/cleanup.sh

```diff
@@ -105,6 +95,19 @@ delete_tvk_op() {
       fi
     fi
   fi
+
+  # Delete k8s-triliovault-resource-cleaner cronjob
+  tvkcron=$(kubectl get cronjob --no-headers -A 2>/dev/null | grep k8s-triliovault | awk '{print $2}')
+  tvkcron_ns=$(kubectl get cronjob --no-headers -A 2>/dev/null | grep k8s-triliovault | awk '{print $1}')
```

Comment:
same here `kubectl get cronjob --no-headers -A 2>/dev/null | grep k8s-triliovault` is common in both commands

Reply:
Done